doc/gradle.md

```diff
@@ -0,0 +1,14 @@
+# Custom gradle parameters
+
+You can use these parameters by specifying them inside the `gradle.properties` file as 
+`systemProp.<name>=<value>` or passing them through the CLI with `-D<name>=<value>`.
+
+## packageSuffix
+This allows you to specify a suffix, which will be added on release builds to the application id, 
+the `app_name` string and the apk file.  
+No validation is made, so make sure to pass in a valid value.
```

Comment:
I would add that by default the git branch name is used, but this can be used to override it

Reply:
This option only overrides the name of release builds, not debug build

> This allows you to specify a suffix, which will be added on >>release<< builds to the application id

Reply:
Ok, then this looks good, sorry for the misunderstanding